### PR TITLE
Check where set file type fails

### PR DESCRIPTION
**Pull request by tzheleznyak**
Test Fix adding of `--output` flag

### CODE REVIEW

File: features/package.feature [hunk 1]
```diff
@@ -6,6 +6,7 @@ Feature: Packaging
     # We use version 0.0.1-suffix to verify that RPM converts dashes to underscores
     # in the version as we expect
     Given I successfully run `env DEBUG=true GLI_DEBUG=true debify package -d ../../example -v 0.0.1-suffix example -- --post-install /distrib/postinstall.sh`
+      And I successfully run `env DEBUG=true GLI_DEBUG=true debify package -d ../../example --output rpm  -v 0.0.1-suffix example  -- --post-install /distrib/postinstall.sh`
```
Comment (orenbm) [inline features/package.feature]: fix indentation (`And` should be aligned with `Given`)

File: Rakefile [hunk 2]
```diff
@@ -33,7 +33,6 @@ if cucumber?
   Cucumber::Rake::Task.new(:features) do |t|
     opts = "features --format junit -o #{CUKE_RESULTS} --format pretty -x"
     opts += " --tags #{ENV['TAGS']}" if ENV['TAGS']
-    opts += " --tags ~@skip"
```
Comment (orenbm) [inline Rakefile]: the better fix IMO would be to remove the `skip` tag from the test that was skipped and leave this.

Comment (orenbm) [reply Rakefile]: for future use

Comment (orenbm) [reply Rakefile]: i would leave this for future use

Comment (tzheleznyak) [reply Rakefile]: Done

**Review by orenbm — APPROVED** [review]

**Pull request merged**